operating/verify-image-signatures.adoc

```diff
@@ -0,0 +1,36 @@
+[id="verify-image-signatures"]
+= Verifying image signatures
+include::modules/common-attributes.adoc[]
+:context: verify-image-signatures
+
+toc::[]
+
+[role="_abstract"]
+You can use {product-title} ({product-title-short}) to ensure the integrity of the container images in your clusters by verifying image signatures against pre-configured keys.
+
+You can create policies to block unsigned images and images that do not have a verified signature.
+You can enforce the policy by using an admission controller to stop unauthorized deployment creation.
+
+[NOTE]
+====
+* {product-title-short} 3.70 only supports Cosign signatures and Cosign public key signature verification. For more information about Cosign, see link:https://docs.sigstore.dev/cosign/overview[Cosign overview].
+* You must configure signature integration with at least 1 Cosign public key for signature verification.
+* For all deployed and watched images:
+** {product-title-short} fetches and verifies the signatures every 4 hours.
+** {product-title-short} verifies the signatures whenever you change or update your signature integration public keys.
+====
+
+//Configuring signature integration
+include::modules/configure-signature-integration.adoc[leveloffset=+1]
+
+//Using signature verification
+include::modules/use-signature-verification.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operating/manage-security-policies.adoc#create-policy-from-system-policies-view_manage-security-policies[Creating a security policy from the system policies view]
+* xref:../operating/manage-security-policies.adoc#policy-criteria_manage-security-policies[Policy criteria]
+
+[role="_additional-resources"]
+== Additional resources
+* xref:../operating/use-admission-controller-enforcement.adoc#enable-admission-controller-enforcement[Enabling admission controller enforcement]
```

Comment:
Would it be possible to also add within the `enable-admission-controller-enforcement` a hint that this is required not only for image scanning but also image signature verification?

Right now, I'd be missing the disadvantages of disabling this setting, i.e.:
```
If you enable this setting, Red Hat Advanced Cluster Security for Kubernetes contacts image scanners if scan and signature verification results are not already available, which adds considerable latency.

If you disable this setting, the enforcement decision only considers image scan and signature verification criteria if cached scan and signature verification results are available.
```